<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>902</width>
    <height>641</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
   </font>
  </property>
  <property name="windowTitle">
   <string comment="Yet Another Viewer for ADVLOG.BIN" extracomment="Yet Another Viewer for ADVLOG.BIN">MHDD ADVLOG.BIN Viewer (by @AloneCafe)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>891</width>
      <height>601</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Meta Info (&amp;M)</string>
     </attribute>
     <widget class="QLineEdit" name="line_path">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>30</y>
        <width>531</width>
        <height>21</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>BIN File Name: </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>461</width>
        <height>461</height>
       </rect>
      </property>
      <property name="title">
       <string>File Header Information</string>
      </property>
      <widget class="QPushButton" name="copyAllTextButton">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>420</y>
         <width>141</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Copy all RAW text</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="line_sector_size">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>310</y>
         <width>291</width>
         <height>21</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="line_scan_block_size">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>350</y>
         <width>291</width>
         <height>21</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="line_fw">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>190</y>
         <width>291</width>
         <height>21</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="line_mode">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>110</y>
         <width>291</width>
         <height>21</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>350</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Scan Block Size:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="line_signature">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>30</y>
         <width>291</width>
         <height>21</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="line_ver">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>70</y>
         <width>291</width>
         <height>21</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="line_sn">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>230</y>
         <width>291</width>
         <height>21</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Mode: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>190</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>F / W: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>270</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Sectors: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="line_device">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>150</y>
         <width>291</width>
         <height>21</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="line_sectors">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>270</y>
         <width>291</width>
         <height>21</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>310</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Sector Size:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Signature: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Device:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Version: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>230</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>S / N: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="line_lba_time_entries">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>390</y>
         <width>291</width>
         <height>21</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>390</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>LBA/Time Entries:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="openButton">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>30</y>
        <width>121</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Open BIN... (&amp;N)</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Results List (&amp;R)</string>
     </attribute>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>241</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>All Sectors info with LBA/Time(ms) :</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="LbaTimeWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>451</width>
        <height>531</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="autoScroll">
       <bool>true</bool>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
      <property name="columnCount">
       <number>2</number>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>150</number>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>22</number>
      </attribute>
      <attribute name="verticalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>LBA (sector size)</string>
       </property>
       <property name="foreground">
        <brush brushstyle="NoBrush">
         <color alpha="255">
          <red>85</red>
          <green>85</green>
          <blue>255</blue>
         </color>
        </brush>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Scan Time (ms)</string>
       </property>
       <property name="foreground">
        <brush brushstyle="NoBrush">
         <color alpha="255">
          <red>170</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </property>
      </column>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>20</y>
        <width>381</width>
        <height>161</height>
       </rect>
      </property>
      <property name="title">
       <string>Time Filter Selection</string>
      </property>
      <widget class="QRadioButton" name="radioButton_Lessthan">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>121</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Less than (ms) &lt;</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_Morethan">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>More than (ms) &gt;</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_Between">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>95</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Between (ms)</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_lessthan">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>30</y>
         <width>151</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>9999999</number>
       </property>
       <property name="value">
        <number>150</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_morethan">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>60</y>
         <width>151</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>9999999</number>
       </property>
       <property name="value">
        <number>700</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_betweenfrom">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>90</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>9999999</number>
       </property>
       <property name="value">
        <number>150</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_betweento">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>90</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>9999999</number>
       </property>
       <property name="value">
        <number>700</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>90</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>~</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_All">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>95</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>All</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="applyFilterButton">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>190</y>
        <width>111</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Apply Filter (&amp;F)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="sortLbaButton">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>10</y>
        <width>101</width>
        <height>24</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Sort entries by LBA</string>
      </property>
      <property name="text">
       <string>Sort by LBA (&amp;L)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="sortTimeButton">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>10</y>
        <width>111</width>
        <height>24</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Sort entries by scan time</string>
      </property>
      <property name="text">
       <string>Sort by Time (&amp;T)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="resetDefaultFilterButton">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>190</y>
        <width>141</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Reset default (&amp;All)</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>902</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>File (&amp;F)</string>
    </property>
    <addaction name="actionOpen_O"/>
    <addaction name="actionClose_C"/>
    <addaction name="separator"/>
    <addaction name="actionExit_Program_E"/>
   </widget>
   <widget class="QMenu" name="menu_V">
    <property name="title">
     <string>View (&amp;V)</string>
    </property>
    <addaction name="actionMeta_Info_M"/>
    <addaction name="actionREsults_List_L"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Options (&amp;O)</string>
    </property>
    <widget class="QMenu" name="menuLanguage_L_2">
     <property name="title">
      <string>Language (&amp;L)</string>
     </property>
     <addaction name="actionEnglish"/>
     <addaction name="actionSimplified_Chinese"/>
    </widget>
    <addaction name="menuLanguage_L_2"/>
   </widget>
   <widget class="QMenu" name="menuHelp_H">
    <property name="title">
     <string>Help (&amp;H)</string>
    </property>
    <addaction name="actionProject_Homepage_H"/>
    <addaction name="actionVisit_me_GitHub_V"/>
    <addaction name="separator"/>
    <addaction name="actionAbout_A"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_V"/>
   <addaction name="menu"/>
   <addaction name="menuHelp_H"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string comment="Designed &amp; Programed By @AloneCafe" extracomment="Designed &amp; Programed By @AloneCafe">Designed &amp; Programed By @AloneCafe</string>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="statusTip">
    <string>Designed &amp; Programed By @AloneCafe</string>
   </property>
  </widget>
  <action name="actionOpen_O">
   <property name="text">
    <string>Open... (&amp;O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionClose_C">
   <property name="text">
    <string>Close &amp;&amp; Refresh (&amp;C)</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>-</string>
   </property>
  </action>
  <action name="actionExit_Program_E">
   <property name="text">
    <string>Exit Program (&amp;E)</string>
   </property>
  </action>
  <action name="actionVisit_me_GitHub_V">
   <property name="text">
    <string>Visit me @GitHub (&amp;V)</string>
   </property>
  </action>
  <action name="actionAbout_A">
   <property name="text">
    <string>About... (&amp;A)</string>
   </property>
  </action>
  <action name="actionSimplified_Chinese">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Simplified Chinese</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionProject_Homepage_H">
   <property name="text">
    <string>Project Homepage (&amp;H)</string>
   </property>
  </action>
  <action name="actionMeta_Info_M">
   <property name="text">
    <string>Meta Info (&amp;M)</string>
   </property>
  </action>
  <action name="actionREsults_List_L">
   <property name="text">
    <string>Results List (&amp;R)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
